src/data.ts

```diff
@@ -36,48 +36,44 @@ export class Data {
     return new Promise((resolve, reject) => {
       chrome.storage.local.get([this.username], (result) => {
         // If the data isn't there, result will be an empty object
-        if (Object.keys(result).length > 0) {
-          // We have a cached object, so check time of cache
-          const cachedData = result[this.username];
-          if (new Date().valueOf() - cachedData.cachedAt < CACHE_THRESHOLD) {
-            // We can use the cached version
-            resolve(cachedData);
-          }
+        if (Object.keys(result).length < 1) {
+          // If we get to this point, there was nothing in cache or the cache was invalid
+          reject();
```

Comment:
Also need to `reject` if the cache is outdated

Do promises auto reject if they reach the end and don't resolve?